releasenotes/notes/move-es-version-vars-17c22cfd558a68d5.yaml

```diff
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    The ``elasticsearch_version`` and ``elasticsearch_reindex_version``
+    variables have been moved to ``user_rpco_variables_defaults.yml``
```

Comment:
I don't think that this is accurate any more? It's been moved into the group_vars.